pkg/ingester/instance.go

```diff
@@ -584,12 +584,17 @@ func (i *instance) GetStats(ctx context.Context, req *logproto.IndexStatsRequest
 				chkFrom, chkThrough := chk.chunk.Bounds()
 
 				if !chk.flushed.Equal(zeroValueTime) && from.Before(chkThrough) && through.After(chkFrom) {
```

Comment:
double checking: shouldn't here be `from.Before(chkThrough) || through.After(chkFrom)` as it is a partial overlap?
Ex:
```
	//          From           chkThrough       Through  chkFrom
	//       ┌────────┬───────────────┬────────┐
	//       │                      |  *      Chunk          *        │
	//       └────────┴──────────────   ┴────────┘
```

Reply:
`from.Before(chkThrough) && through.After(chkFrom)` can only be true when there's some degree of overlap with the time range and chunk. In your ascii example that holds true. If we used `||` instead of `&&`, it would include potentially non-overlapping ranges.